examples/ovault-evm/deploy/MyERC4626.ts

```diff
@@ -4,15 +4,13 @@ import { type DeployFunction } from 'hardhat-deploy/types'
 
 import { assetOFTContractName } from './MyAssetOFT'
 
-export const ovaultContractName = 'MyOVault'
-const tokenName = 'MyMockShareOFT'
-const tokenSymbol = 'MockShare'
+export const ovaultContractName = 'MyERC4626'
+const tokenName = 'MyShareOFT'
+const tokenSymbol = 'SHARE'
 
```

Comment:
```suggestion
const export tokenName = 'MyShareOFT'
const export tokenSymbol = 'SHARE'
```

since `asset` and `share` are re-used across different files. i dont like the user having to re-define these things. what do you think about creating a new file here called `config.ts` where we add

`assetToken, shareToken, etc` as types `{name: string, symbol: string}` and pass the objects to these 3 deploy scripts?